src/components/containers/PanelHeader.js

```diff
@@ -43,10 +55,29 @@ class PanelHeader extends Component {
               <Button
                 variant="primary"
                 className="js-add-button"
-                onClick={() => addAction.handler(this.context)}
+                onClick={
+                  Array.isArray(addAction.handler)
+                    ? this.togglePanel
+                    : () => addAction.handler(this.context)
+                }
                 icon={icon}
                 label={addAction.label}
-              />{' '}
+              />
```

Comment:
this allows for dropdowns on +... needs better styling!

Reply:
Can you make an issue for this with a screenshot and @aulneau can apply his magic

Reply:
yes once merged

Reply:
#439